src/redisraft.c

```diff
@@ -555,6 +555,66 @@ static void handleAsking(RedisRaftCtx *rr, RedisModuleCtx *ctx)
     RedisModule_ReplyWithSimpleString(ctx, "OK");
 }
 
+static void handleClientUnblock(RedisRaftCtx *rr, RedisModuleCtx *ctx, RaftRedisCommand *cmd)
+{
+    raft_session_t id;
+    bool error = false;
+
+    if (cmd->argc != 3 && cmd->argc != 4) {
+        RedisModule_ReplyWithError(ctx, "ERR syntax error");
+        return;
+    }
+
+    int ret = RedisModule_StringToULongLong(cmd->argv[2], &id);
+    if (ret != REDISMODULE_OK) {
+        RedisModule_ReplyWithError(ctx, "ERR value is not an integer or out of range");
+        return;
+    }
+
+    if (cmd->argc == 4) {
+        size_t str_len;
+        const char *str = RedisModule_StringPtrLen(cmd->argv[3], &str_len);
+
+        if (!strcasecmp(str, "timeout")) {
+            error = false;
+        } else if (!strcasecmp(str, "error")) {
```

Comment:
probably we should use strncasecmp for these

Reply:
yes

Reply:
well, redis does strcasecmp().